sechub-pds-solutions/phan/docker/Phan-Debian.dockerfile

```
@@ -0,0 +1,56 @@
+# SPDX-License-Identifier: MIT
+
+# The image argument needs to be placed on top
+ARG BASE_IMAGE
+FROM ${BASE_IMAGE}
+
+# The remaining arguments need to be placed after the `FROM`
+# See: https://ryandaniels.ca/blog/docker-dockerfile-arg-from-arg-trouble/
+
+LABEL org.opencontainers.image.source="https://github.com/mercedes-benz/sechub"
+LABEL org.opencontainers.image.title="SecHub Phan+PDS Image"
+LABEL org.opencontainers.image.description="A container which combines Phan Wrapper with the SecHub Product Delegation Server (PDS)"
+LABEL maintainer="SecHub FOSS Team"
+
+USER root
+
+# Build Args
+ARG PHAN_WRAPPER_VERSION="5.4.3"
```

A wrapper is only necessary in case of cloud services/proprietary tools which cannot run locally. Phan can run inside the container just fine.

```suggestion
ARG PHAN_VERSION="5.4.3"
```

---

Done.